src/client/src/tscSql.c

```diff
@@ -416,6 +416,13 @@ int taos_affected_rows(TAOS_RES *tres) {
   return pSql->res.numOfRows;
 }
 
+int taos_affected_tables(TAOS_RES *tres) {
```

Comment:
why is this API needed?

Reply:
Deleted 1000 row(s) from 1000 table(s) (0.331877s) .  1000 tables can be got by taos_affected_tables() api